package.json

```diff
@@ -58,8 +58,8 @@
     "luxon": "^3.5.0",
     "marked": "^14.1.2",
     "nanoid": "^5.0.7",
-    "ol": "^10.2.1",
-    "proj4": "^2.12.1",
+    "ol": "^10.3.1",
```

Comment:
I have opened https://github.com/openlayers/openlayers/issues/16436 to report the `ReferenceError: Worker is not defined` error message